ppr-ui/tests/unit/TombstoneDischarge.spec.ts

```diff
@@ -58,28 +58,35 @@ describe('Tombstone component', () => {
   }
   const registrationType = mockedSelectSecurityAgreement()
 
-  beforeEach(async () => {
-    // mock the window.location.assign function
-    delete window.location
-    window.location = { assign: jest.fn() } as any
-
+  beforeAll(async () => {
     // setup data
     await store.dispatch('setRegistrationType', registrationType)
     await store.dispatch('setRegistrationNumber', registration.baseRegistrationNumber)
     await store.dispatch('setRegistrationCreationDate', registration.createDateTime)
     await store.dispatch('setRegistrationExpiryDate', registration.expiryDate)
   })
 
+  beforeEach(async () => {
+    // mock the window.location.assign function
+    delete window.location
+    window.location = { assign: jest.fn() } as any
+  })
+
   afterEach(() => {
     window.location.assign = assign
     wrapper.destroy()
   })
 
+  afterAll(async () => {
+    await store.dispatch('setRegistrationType', null)
+    await store.dispatch('setRegistrationNumber', null)
+    await store.dispatch('setRegistrationCreationDate', null)
+    await store.dispatch('setRegistrationExpiryDate', null)
+  })
+
   it('renders Tombstone component properly for Total Discharge', async () => {
-    await store.dispatch('setMhrInformation', mockedMhrInformation)
```

Comment:
I don't believe the delete lines do anything in the test, specifically cause the code does not call Vue.nextTick, and the tests seem to be meant for PPR.